WordPress/Classes/ViewRelated/Plans/PlanDetailViewController.swift

```diff
@@ -247,6 +226,65 @@ class PlanDetailViewController: UIViewController {
             alert.presentFromRootViewController()
         }
     }
+
+    // MARK: - NoResults Handling
+
+    private func setupNoResultsViewController() {
+        let noResultsStoryboard = UIStoryboard(name: "NoResults", bundle: nil)
+        guard let noResultsViewController = noResultsStoryboard.instantiateViewController(withIdentifier: "NoResults") as? NoResultsViewController else {
+            return
+        }
+
+        noResultsViewController.delegate = self
+        self.noResultsViewController = noResultsViewController
+    }
+
+    private func updateNoResults() {
+        hideNoResults()
+        if let noResultsViewModel = viewModel.noResultsViewModel {
+            showNoResults(noResultsViewModel)
+        }
+    }
+
+    private func showNoResults(_ viewModel: NoResultsViewController.Model) {
+
+        if noResultsViewController == nil {
+            setupNoResultsViewController()
+        }
+
+        guard let noResultsViewController = noResultsViewController else {
+            return
+        }
+
+        noResultsViewController.bindViewModel(viewModel)
+
+        noResultsViewController.view.frame = noResultsViewFrame()
+        noResultsViewController.view.backgroundColor = UIColor.white
```

Comment:
Small detail: `UIColor` can 💥 for less verbosity

Reply:
Ya know, I know that. And one day I'll stop doing it. 😉